Client/Telemetry/TelemetryWrapper.swift

```diff
@@ -196,20 +195,6 @@ class TelemetryWrapper: TelemetryWrapperProtocol, FeatureFlaggable {
         )
     }
 
-    // Sets hashed fxa sync device id for glean deletion ping
-    func setSyncDeviceId() {
-        // Grab our token so we can use the hashed_fxa_uid and clientGUID for deletion-request ping
-        guard let accountManager = RustFirefoxAccounts.shared.accountManager.peek(),
-              let state = accountManager.deviceConstellation()?.state(),
-              let clientGUID = state.localDevice?.id
-        else { return }
-
-        RustFirefoxAccounts.shared.syncAuthState.token(Date.now(), canBeExpired: true) >>== { (token, _) in
```

Comment:
Wonderful! I had noticed some issues related to this code [here](https://github.com/mozilla-mobile/firefox-ios/issues/17075). I just closed the task 😄 Thank you

Reply:
yes! I realized that as well when I was trying to refactor that code, it's also not implemented on the android side -- so it'd be better to remove it here and come up with an implemented for both consumers!